packages/react-component-library/src/components/TabSet/TabSet.test.tsx

```diff
@@ -363,4 +363,22 @@ describe('TabSet', () => {
       expect(wrapper.getByTestId('scroll-right')).toBeInTheDocument()
     })
   })
+
+  describe('when the tab set hasOverflow', () => {
+    beforeEach(() => {
+      wrapper = render(
+        <TabSet hasOverflow>
+          <TabSetItem title="Title 1">Content 1</TabSetItem>
+          <TabSetItem title="Title 2">Content 2</TabSetItem>
+          <TabSetItem title="Title 3">Content 3</TabSetItem>
+        </TabSet>
+      )
+    })
+
+    it('should have overflow visible', () => {
+      expect(wrapper.getByTestId('tabset-styledbody')).toHaveStyleRule(
+        'overflow', 'visible',
```

Comment:
Maybe test for the inverse eg `should not have overflow visible when `hasOverflow` is `false``